### PR TITLE
✨ Load a wiki page after Selenium is initialized to remind the user to look back at the terminal to answer more questions

### PR DESCRIPTION
**Pull request by coderanger**
# Description

I set the page to `Desktop-Tool-Landing` but it can be whatever. Some example content (because wikis don't accept PRs):

```md
# Congratulations!

MPCAutofill has started successfully, please look back at your terminal window to answer a few more questions before the upload can begin.

Check out [the documentation if you have any further questions](Desktop-Tool).
```

# Checklist

- [x] I have installed `pre-commit` and installed the hooks with `pre-commit install` before creating any commits.
- [ ] I have updated any related tests for code I modified or added new tests where appropriate.
- [x] I have manually tested my changes as follows:
  - Ran autofill.py and observed it working.
- [ ] I have updated any relevant documentation or created new documentation where appropriate.

### CODE REVIEW

**Comment by ndepaola** [comment]
thanks for getting the ball rolling on this! i've been meaning to make this change for a few weeks but didn't get around to it.

after some more thought, i think it would be better to load a local HTML page rather than loading a page hosted in the wiki - a) because it means the page should load instantly, and b) because it doesn't really belong in the wiki.

would you mind taking a look at whether it's possible to point selenium at a local HTML file? you may also need to update the pyinstaller build to include the new file

**Comment by coderanger** [comment]
Surely, if you want to go even further it could be a localhost webserver and move the config from terminal into a UI? That might be too much though :)

**Comment by coderanger** [comment]
I split the difference, it's setup on an internal web server but just a single static page for now.

**Comment by coderanger** [comment]
Is the test failure real? Seems unrelated in the PDF functionality and those tests haven't been run in a while. I can look tomorrow or something.

**Comment by ndepaola** [comment]
> Surely, if you want to go even further it could be a localhost webserver and move the config from terminal into a UI? That might be too much though :)

we certainly could do yea - though i think it's out of scope here. let's roll with the local webserver approach you have for now and come back to reworking the UI later

> Is the test failure real? Seems unrelated in the PDF functionality and those tests haven't been run in a while. I can look tomorrow or something.

it looks like this is because you haven't set up your repo fork with google drive credentials. this is touched on here https://github.com/chilli-axe/mpc-autofill/wiki#contributing but it looks like the docs are a bit out of date - this now matters when modifying desktop tool code as well as backed code. i'll update the docs in a sec to clarify this

File: desktop-tool/src/web_server.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+class _Handler(server.BaseHTTPRequestHandler):
+    def do_GET(self):
+        self.send_response(200)
+        self.send_header("Content-Type", "text/html")
+        self.end_headers()
+        self.wfile.write(
+            Path(__file__).joinpath("../..").joinpath(constants.POST_LAUNCH_HTML_FILENAME).resolve().read_bytes()
+        )
+
+    def log_request(self, code: int | str = "-", size: int | str = "-") -> None:
```
Comment (ndepaola) [inline desktop-tool/src/web_server.py:18]: to maintain compatibility with windows 7 for as long as possible, we're still on python 3.9 - so you'll need to use old-style typing here:

```suggestion
    def log_request(self, code: Union[int, str] = "-", size: Union[int, str] = "-") -> None:
```

**Review by ndepaola — COMMENTED** [review]
lgtm pending my one comment, thanks for this PR!

**Comment by ndepaola** [comment]
merging w/ unresolved comment, i'll fix this bit up

**Pull request merged**